gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -272,10 +282,14 @@ def setup_ssh_cluster_with_retries(self) -> SSHCluster:
             )
         return cluster
 
-    def run(self) -> GtsfmData:
+    def run(self, graph_partitioner: GraphPartitionerBase = None) -> GtsfmData:
```

Comment:
nit: we dont need this argument, for now you can use the single partitioner as is. Once metis is ready, we can select the partitioner using our configuration system (the same way we do for other gtsfm modules).